File: Assetmanagement/Assetmanagement/Models/MaterializePOModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Assetmanagement.Models
{
    public class MaterializePOModel
    {
        public long AssetQuotationId { get; set; }
        public Nullable<long> AssetRequestId { get; set; }
        public long AssetTypeId { get; set; }
        public string AssetTypeName { get; set; }
        public string VendorName { get; set; }
        public string AssetStatusName { get; set; }
        public int ApprovedBy { get; set; }
        public DateTime ApprovedOn { get; set; }
        public string EmployeeName { get; set; }
    }
}

File: AMS_Source/src/app/views/assetrequests/materializepos/shared/materializepo.model.ts
export class MaterializePOModel {
    AssetQuotationId: string;
    AssetRequestId: string;
    AssetTypeId: string;
    AssetTypeName: string;
    VendorName: string;
    AssetStatusName: string;
    ApprovedBy: string;
    ApprovedOn: string;
    EmployeeName: string;
    Location:string;
}


export class AssetRequestPaymentsModel{
    AssetRequestId: string;
    PaymentModeName: string;
    ChequeAmount: string;
    CreditCardNo: string;
    CreditCardOwner: string;
    ChequeNo: string;
    ChequeDate: string;
    ChequeBankName: string;
    ChequeBranch: string;
    DC_TransactionIdentifier: string;
    DC_Transferer: string;
}

export class PaymentDetails{
    PaymentMode:string;
    BaseAmount: string;
    Tax: string;
    PaymentId: string;
    ARNumber: string;
    AssetType: string;
    RequestedBy: string;
    CreatedOn: string;
}

export class OtherAssetTypeSpecification{
    AssetTypeOtherSpecification: string;
}

export class AssetTypePaymentBalance{
    Status: string;
    BalanceAmount: string;
}
